recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,147 @@
+        self.lang = lang
+        self.url = url
+        message = f'Detected recipe locale "{self.lang}" is not supported for url={url}'
+        super().__init__(message)
+
+
+class MarleySpoon(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"marleyspoon.{domain}"
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"})["content"]
+
+    def author(self):
+        return normalize_string(self.soup.find(class_="author-info").a.get_text())
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h3", {"class": "recipe-title"}).get_text()
+        )
+
+    def _specs(self):
+        vocab = {
+            "de": {
+                "time": "Dauer",
+                "category": "Schwierigkeitsgrad",
+                "nutrients": "Nährwertangaben",
+            },
+            "nl": {
+                "time": "Bereidingstijd",
+                "category": "Niveau",
+                "nutrients": "Voedingswaarde",
+            },
+            "en": {
+                "time": "Serving Time",
+                "category": "Level",
+                "nutrients": "Nutrition",
+            },
+        }
+
+        self.locale = self.language()[:2].lower()
+        if self.locale not in vocab.keys():
+            raise UnsupportedLocale(self.locale)
+
+        recipe_specs = self.soup.find("ul", {"class": "recipe-specs"})
+
+        time_label = recipe_specs.find("label", string=vocab[self.locale]["time"])
+        category_label = recipe_specs.find(
+            "label", string=vocab[self.locale]["category"]
+        )
+        nutrients_pattern = "^{}.*$".format(vocab[self.locale]["nutrients"])
+        nutrients_label = recipe_specs.find(
+            "label", string=re.compile(nutrients_pattern)
+        )
+
+        time_info = time_label.find_next_sibling("p").get_text()
+        category_info = category_label.find_next_sibling("p").get_text()
+        nutrients_info = nutrients_label.find_next_sibling("p").get_text()
+
+        return time_info, category_info, nutrients_info
+
+    def total_time(self):
+        return get_minutes(self._specs()[0])
+
+    def category(self):
+        return normalize_string(self._specs()[1])
+
+    def nutrients(self):
+        nutrients = {}
+
+        vocab = {
+            "calories": {
+                "de": "Kalorien",
+                "nl": "Calorieën",
+                "en": "Calories",
+                "unit": "calories",
+            },
+            "fatContent": {"de": "Fett", "nl": "Vet", "en": "Fat", "unit": "grams fat"},
+            "proteinContent": {
+                "de": "Eiweiß",
+                "nl": "Eiwit",
+                "en": "Proteins",
+                "unit": "grams protein",
+            },
+            "carbohydrateContent": {
+                "de": "Kohlenhydrate",
+                "nl": "Koolhydraten",
+                "en": "Carbs",
+                "unit": "grams carbohydrates",
```

Comment:
On second thoughts.. let's omit the nutrient name from the `unit` field for each of these.  It is redundant, given that the dictionary returned by the `nutrients` method includes the names in the keys.

```suggestion
            "fatContent": {"de": "Fett", "nl": "Vet", "en": "Fat", "unit": "grams"},
            "proteinContent": {
                "de": "Eiweiß",
                "nl": "Eiwit",
                "en": "Proteins",
                "unit": "grams",
            },
            "carbohydrateContent": {
                "de": "Kohlenhydrate",
                "nl": "Koolhydraten",
                "en": "Carbs",
                "unit": "grams",
```

Reply:
I thought it was redundant too, but I was following this example:
https://github.com/hhursev/recipe-scrapers/blob/main/docs/in-depth-guide-scraper-functions.md#nutrients---dictstr-str